modules/admin_manual/pages/enterprise/authentication/kerberos.adoc

```diff
@@ -296,7 +307,7 @@ ktpass
 Note that the parameter https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/ktpass[crypto,window=_blank] is according the Microsoft documentation recommended to be set.
 --
 
-. Move the generated keytab file `<user-name>.keytab` to the Linux server hosting the ownCloud instance to location `<keytab-file-location>`.
+. Move the generated keytab file `<user-name>.keytab` to the Linux server hosting the ownCloud instance to location `<keytab-file-location>`. Note that the file must be accessible by the web server.
```

Comment:
Maybe we should add a note that this file should not be place in the Apache root dir so an attacker could download it if the Apache redirect rules are wrong. To make things clear: If the keytab file gets into the wrong hand the attacker can impersonate himself as any user for which delegation is allowed.

Reply:
There is a block below that the keytab file on the ownCloud server must be properly protected. We can sharpen it there.